c2rust-transpile/src/translator/named_references.rs

```diff
@@ -34,6 +34,21 @@ fn is_simple_lvalue(e: &Box<Expr>) -> bool {
     }
 }
 
+pub struct NamedReference<R> {
+    pub lvalue: Box<Expr>,
+    pub rvalue: R,
+}
+
+impl<R> NamedReference<R> {
+    pub fn map<S, F: Fn(R) -> S>(self, f: F) -> NamedReference<S> {
```

Comment:
Rename this to `map_rhs`.

Reply:
I renamed it to `map_rvalue` to be consistent with the field name.